gpt_index/embeddings/openai.py

```diff
@@ -69,6 +70,15 @@ class OpenAIEmbeddingMode(str, Enum):
 }
 
 
+@retry(wait=wait_random_exponential(min=1, max=20), stop=stop_after_attempt(6))
+def get_embedding(
```

Comment:
This works. Would you be able to add a comment (or a docstring) referencing the original [file link](https://github.com/openai/openai-python/blob/main/openai/embeddings_utils.py)? Just so we know it's copied.